File: PPFUV/PPFUV/Model/Pozoriste.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PPFUV.Model
{
    public class Pozoriste
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int id { get; set; }

        public int brZaposlenih { get; set; }

        public string naziv { get; set; }

        public ICollection<Sala> sale { get; set; }

        //public OrgOdb orgOdb { get; set; }

        //public ICollection<Izabran> izabrani { get; set; }
    }
}

File: PPFUV_front/bp2/src/app/entities/pozoriste/pozoriste.ts
import { Festival } from '../festival/festival';
import { PropDeoFest } from '../prop-deo-fest/prop-deo-fest';
import { OrgOdb } from '../org-odb/org-odb';
import { Sala } from '../sala/sala';
import { Izabran } from '../izabran/izabran';

export class Pozoriste {
    id: number;
    brZaposlenih: number;
    naziv: string;
    
    // festivali: Array<Festival>;
    // propDeloviFest: Array<PropDeoFest>;
    orgOdb: OrgOdb;
    sale: Array<Sala>;

    // izabrani: Array<Izabran>;
}
